annotation-web/src/main/java/eu/europeana/annotation/web/service/impl/BaseAnnotationValidator.java

```diff
@@ -378,7 +385,7 @@ public void validateAnnotationId(AnnotationId annoId) throws ParamValidationExce
      * @throws RequestBodyValidationException
      */
     protected void validateEdmRights(Body body) throws ParamValidationException, RequestBodyValidationException {
-
+    if(body==null) return;
```

Comment:
if the argument is null, that is an error case that cannot be ignored. To avoid NPE, a PropertyValidationException must be thrown

Reply:
Improved.